docs/enums/DefaultConnectionType.html

```diff
@@ -86,7 +86,7 @@ <h3>Proxy<wbr>Actions<wbr>Only</h3>
 					<div class="tsd-signature tsd-kind-icon">Proxy<wbr>Actions<wbr>Only<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> = 3</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/coda/packs-sdk/blob/main/types.ts#L46">types.ts:46</a></li>
+							<li>Defined in <a href="https://github.com/kr-project/packs-sdk/blob/main/types.ts#L46">types.ts:46</a></li>
```

Comment:
Hmm, this diffs are undesirable. I think you may need to update your remote to point to the new `coda` org in the url?

Reply:
good find.

Reply:
Alternatively you could maybe add the `--gitRemote` flag to the `typedoc` target in the Makefile so it's not dependent on any developer's particular git configuration.

Reply:
good catch. this is so easy to be ignored. let me add a linter rule I think ..